packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx

```diff
@@ -114,17 +115,7 @@ export class PersonaCoin extends React.Component<IPersonaProps, IPersonaState> {
                 </div>
               )
             }
-            <Image
-              className={ css('ms-Persona-image', styles.image) }
-              imageFit={ ImageFit.cover }
-              src={ imageUrl }
-              width={ coinSize || SIZE_TO_PIXELS[size] }
-              height={ coinSize || SIZE_TO_PIXELS[size] }
-              alt={ imageAlt }
-              shouldFadeIn={ imageShouldFadeIn }
-              shouldStartVisible={ imageShouldStartVisible }
-              onLoadingStateChange={ this._onPhotoLoadingStateChange }
-            />
+            { this._onRenderCoin( this.props, size ) }
```

Comment:
`onRenderCoin` could be refactored a bit. `IRenderFunction` should take `props` and the default render as parameters. See [Tooltip](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.tsx)'s `onRenderContent` for example

Reply:
Ah nice! yeah that's much cleaner thank you so much!